infrastructure/helm/quantumserverless/templates/raysystemmetricsdashboard.yaml

```diff
@@ -0,0 +1,660 @@
+{{- if .Values.prometheusEnable }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    grafana_dashboard: "1"
+    release: {{ .Release.Name }}
+  name: ray-system-metrics-dashboard
+  namespace: {{ .Release.Namespace }}
+data:
+  quantum-serverless.json: |-
```

Comment:
Would you change this file name?  All the dashboard json files are put into the same directory.  This file name conflicts with an already existing one.